src/Symfony/Component/Polyfill/Tests/IconvTest.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Polyfill\Tests;
+
+use Symfony\Component\Polyfill\Iconv as p;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ *
+ * @covers Symfony\Component\Polyfill\Iconv::<!public>
+ * @requires extension iconv
+ */
+class IconvTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @covers Symfony\Component\Polyfill\Iconv::iconv
+     */
+    public function testIconv()
+    {
+        // Native iconv() behavior varies between versions and OS for these two tests
+        // See e.g. https://bugs.php.net/52211
+        $this->assertFalse(p::iconv('UTF-8', 'ISO-8859-1', 'nœud'));
+        $this->assertSame('nud', p::iconv('UTF-8', 'ISO-8859-1//IGNORE', 'nœud'));
+
+        $this->assertSame(utf8_decode('déjà'), p::iconv('CP1252', 'ISO-8859-1', utf8_decode('déjà')));
+        $this->assertSame('déjà', p::iconv('UTF-8', 'utf8', 'déjà'));
+        $this->assertSame('deja noeud', p::iconv('UTF-8', 'US-ASCII//TRANSLIT', 'déjà nœud'));
+
+        $this->assertSame('4', p::iconv('UTF-8', 'UTF-8', 4));
+    }
+
+    /**
+     * @covers Symfony\Component\Polyfill\Iconv::iconv_strlen
+     * @covers Symfony\Component\Polyfill\Iconv::strlen1
+     * @covers Symfony\Component\Polyfill\Iconv::strlen2
```

Comment:
yes I know but this imports a code that existed previously, and removing the `@covers` annotation would only reduce the quality of the code (when running coverage tests). Which means I'd like we keep them.